recipes-devtools/dediprog-flasher/dediprog-flasher.bb

```diff
@@ -0,0 +1,21 @@
+DESCRIPTION = "Linux software for Dediprog SF100 and SF600 SPI flash programmers"
+SECTION = "devel"
+LICENSE = "GPL-2.0"
+LIC_FILES_CHKSUM = "file://LICENSE;md5=a23a74b3f4caf9616230789d94217acb"
+
+DEPENDS = "libusb"
+
+SRCREV = "e691f2d432144e3dbc82e9e0eea1ebaed4f3becf"
+SRC_URI = "git://github.com/DediProgSW/SF100Linux.git;protocol=https"
+
+PV = "1.0+${SRCPV}"
+
+S = "${WORKDIR}/git"
+
+do_install () {
+    oe_runmake DESTDIR=${D} install
+}
+
+BBCLASSEXTEND = "native nativesdk"
```

Comment:
this is not needed

Reply:
[Adressed](https://github.com/3mdeb/meta-rte/pull/69/commits/78e7d6769dd5efc4ea92c711cfdf56382e75c582#diff-9636c207549f3e079037ebd1febc2ab61270d8a4787831176a23b32ec1028d77R17-R19)